content/docs/0.6.0/usecases/self-healing-with-keptn/dynatrace-unleash/index.md

```diff
@@ -9,7 +9,7 @@ Demonstrates how to use the self-healing mechanisms of Keptn to automatically se
 
 ## About this tutorial
 
-In this tutorial, you will learn how to use the capabilities of Keptn to provide self-healing for an application with feature flags based on the [Unleash feature toggle framework](https://unleash.github.io/).
+In this tutorial, you will learn how to use the capabilities of Keptn to provide self-healing for an application with feature flags based on the [Unleash feature toggle framework](https://unleash.github.io/). Please note that the _carts_ microservice that we used earlier is already prepared to work with feature toggles. 
```

Comment:
Please rephrase it as follows:

Please note that within this tutorial we use the _carts_ microservice (see prerequesits below) which is already prepared to work with feature toggles.

Reply:
Thanks for your feedback. I've updated the paragraph.